src/components/card/request-cards.ts

```diff
@@ -0,0 +1,53 @@
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import { Mode, themeCSS } from '../../styles/theme'
+import '../card/request-card'
+import { ExplorerConfig, RequestItem } from '../../_types'
+import { defaultExplorerConfig } from '../../helpers/default-explorer-config'
+
+@customElement('radix-request-cards')
+export class RadixRequestCards extends LitElement {
+  @property({
+    type: String,
+    reflect: true,
+  })
+  mode: Mode = 'light'
+
+  @property({ type: Array })
+  requestItems: RequestItem[] = []
+
+  @property({ type: Object })
+  explorer: ExplorerConfig = defaultExplorerConfig
```

Comment:
maybe we should skip adding `ExplorerConfig` into connect button and instead dispatch an event which is handled in RDT

Reply:
this would be for both accounts & transactions. I'm fine with that change

Reply:
up to you